### PR TITLE
Add stop after each song repeat mode

### PR DESCRIPTION
**Pull request by kiselev-dv**
It's very useful to have such mode for concerts or performances when you have ordered list of tracks and starts them as performance go.

### CODE REVIEW

**Comment by kiselev-dv** [comment]
There is a strange behavior, which I didn't get.
In player.cpp at line 295 we call:
```
app_->playlist_manager()->active()->set_current_row(-1);
```
Why? This call confuses autoselection of next row in playlist after stop_after have been called.

File: src/widgets/osd.cpp [hunk 1]
```diff
@@ -314,6 +314,9 @@ void OSD::RepeatModeChanged(PlaylistSequence::RepeatMode mode) {
       case PlaylistSequence::Repeat_Playlist:
         current_mode = tr("Repeat playlist");
         break;
+      case PlaylistSequence::Repeat_OneByOne:
+        current_mode = tr("Tracks one by one");
```
Comment (hatstand) [inline src/widgets/osd.cpp]: I think "Stop after every track" is more descriptive

Comment (kiselev-dv) [inline src/widgets/osd.cpp]: Mb. "Stop after each track" then?

File: src/core/player.cpp [hunk 2]
```diff
@@ -292,7 +292,7 @@ void Player::RestartOrPrevious() {
 
 void Player::Stop(bool stop_after) {
   engine_->Stop(stop_after);
-  app_->playlist_manager()->active()->set_current_row(-1);
+  //app_->playlist_manager()->active()->set_current_row(-1);
```
Comment (hatstand) [inline src/core/player.cpp]: If you're certain about this, just delete it, don't leave commented out code lying around.

**Comment by kiselev-dv** [comment]
Sorry, I've used comments for testing and forget to delete them.

**Comment by hatstand** [comment]
Please run `make format`

**Comment by kiselev-dv** [comment]
done

**Comment by hatstand** [comment]
This failed to build so I reverted it.

**Pull request merged**